examples/hybrid/sphere/baroclinic_wave_utilities.jl

```diff
@@ -6,15 +6,18 @@ const CM = CloudMicrophysics
 
 include("../staggered_nonhydrostatic_model.jl")
 
-struct BaroclinicWaveParameterSet <: AbstractEarthParameterSet end
+struct BaroclinicWaveParameterSet{NT} <: AbstractEarthParameterSet
+    nt::NT
```

Comment:
while you fix the buildkite failure, can we also think of a better property name here? I do not understand what `nt` referes to

Reply:
Agreed, how about `named_tuple`?